packages/admin-ui/ui/src/components/templates/product-table/use-product-actions.tsx

```diff
@@ -1,17 +1,25 @@
-import { Product } from "@medusajs/medusa"
-import { useAdminDeleteProduct, useAdminUpdateProduct } from "medusa-react"
-import { useNavigate } from "react-router-dom"
-import { useTranslation } from "react-i18next"
-import useImperativeDialog from "../../../hooks/use-imperative-dialog"
-import useNotification from "../../../hooks/use-notification"
-import { getErrorMessage } from "../../../utils/error-messages"
+import {
+  adminInventoryItemsKeys,
+  useAdminDeleteProduct,
+  useAdminUpdateProduct,
+  useMedusa,
+} from "medusa-react"
+
+import { ActionType } from "../../molecules/actionables"
 import DuplicateIcon from "../../fundamentals/icons/duplicate-icon"
 import EditIcon from "../../fundamentals/icons/edit-icon"
+import { Product } from "@medusajs/medusa"
 import PublishIcon from "../../fundamentals/icons/publish-icon"
 import TrashIcon from "../../fundamentals/icons/trash-icon"
 import UnpublishIcon from "../../fundamentals/icons/unpublish-icon"
-import { ActionType } from "../../molecules/actionables"
+import { getErrorMessage } from "../../../utils/error-messages"
 import useCopyProduct from "./use-copy-product"
+import { useFeatureFlag } from "../../../providers/feature-flag-provider"
+import useImperativeDialog from "../../../hooks/use-imperative-dialog"
+import { useNavigate } from "react-router-dom"
+import useNotification from "../../../hooks/use-notification"
+import { useQueryClient } from "@tanstack/react-query"
+import { useTranslation } from "react-i18next"
 
 const useProductActions = (product: Product) => {
```

Comment:
todo: We should also delete inventory items when you delete a product from the details page. This is only for the table action, as far as I can see